lib/ansible/module_utils/facts/system/distribution.py

```diff
@@ -360,10 +360,9 @@ def parse_distribution_file_Debian(self, name, data, path, collected_facts):
             debian_facts['distribution'] = 'Cumulus Linux'
             version = re.search(r"VERSION_ID=(.*)", data)
             if version:
-                major, _minor, _dummy_ver = version.group(1).split(".")
                 debian_facts['distribution_version'] = version.group(1)
-                debian_facts['distribution_major_version'] = major
-
+                for val, name in zip(version.group(1).split("."), ['major', 'minor', 'maintenance', 'special']):
+                    debian_facts['distribution_%s_version' % name] = val
```

Comment:
There are no facts as `distribution_maintenance_version` and `distribution_special_version`. So need to remove them. 

Also, please add a changelog entry [here](https://github.com/ansible/ansible/tree/devel/changelogs/fragments) and test case for this change just like [this](https://github.com/ansible/ansible/blob/devel/test/units/module_utils/facts/system/distribution/fixtures/cumulus_linux_2.5.4.json).